File: java/src/main/java/com/samples/Sample.java
package com.samples;

import java.util.ArrayList;

public class Sample {

    public static void main (String ... args){
        ArrayList<Person> persons = new ArrayList<Person>(){{
            add(new Person("Bert", 32));
            add(new Person("Blaise", 3));
            add(new Person("Caden", 1));
        }};

        Person found = null;
        for(Person p2 : persons){
            if(p2.getName().equals("Bert")){
                found = p2;
            }
        }
        if(found != null ){
           System.out.println("Found " + found.getName()) ;
        }


        Person bert = persons.stream()
                .filter( person -> person.getName().equals("Bert"))
                .findAny()
                .orElse(null);
        System.out.println(bert.getName());
        System.out.println(bert.toString());


    }
}

File: kotlin/src/main/kotlin/com/samples/Sample.kt
package com.samples

object Sample {

    @JvmStatic
    fun main(args: Array<String>) {
        val p = Person("Yo",2)

        val persons = listOf(
                Person("Bert", 32),
                Person("Blaise", 3),
                Person("Caden",1)
        )

        val bert = persons.find{ person -> person.name.equals("Bert") }
        println(bert?.name)
        println(bert.toString())

    }
}
